robot_smach_states/src/robot_smach_states/human_interaction/find_person_in_room.py

```diff
@@ -51,7 +50,7 @@ def execute(self, userdata=None):
         look_distance = 2.0  # magic number 4
         look_angles = [f * math.pi / d if d != 0 else 0.0 for f in [-1, 1] for d in [0, 6, 4, 2.3]]  # Magic numbers
         head_goals = [kdl_conversions.VectorStamped(x=look_distance * math.cos(angle),
-                                                    y=look_distance * math.sin(angle), z=1.7,
+                                                    y=look_distance * math.sin(angle), z=1.3,
```

Comment:
This should be set o the higher goal. 1.3 meters is too low for face detection in most cases.